### DISCUSSION TITLE
Team members remain subscribed after team review requested

### DISCUSSION THREAD

**Discussion by jeffchase** [community/community#53163] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

It looks like when a review is requested from a team that every member is subscribed to the pull request. After someone reviews it and the team is removed as a reviewer each individual remains subscribed to the request. This creates a lot of notification noise and even confusion.

Should each individual team members remain subscribed after a review is made on behalf of the team?

**Comment by smarts** [comment]
Are you using a `CODEOWNERS` file? I participate in a monorepo that uses a `CODEOWNERS` file with multiple github teams referenced within. When a PR is created it will auto-add the various teams based on the ownership of the changed files. When a user from one of the specified owning teams comments, the team is removed and [only] that user is added to the PR. This sounds like what you want, but IMHO I prefer one of the following behaviors (ordered by most preferred first):
1. the team remains on the PR
   1. no users are added
   2. the user who commented is also added
1. the behavior you described, in which the team is removed and each user in the team is added

The point is, we operate in a way were we encourage all members of the team to participate in the PR as opposed to a single team member taking ownership of the code review. I understand how this could undesirable for larger teams, but it would be really nice if this were configurable. I'm curious how you are achieving the described behavior.

The reason it's useful to not remove the team from the PR is that it will remain in the Review requested view for all the members of the github team: `https://github.com/pulls/review-requested`

**Reply by jeffchase** [reply]
I don't think the repo uses `CODEOWNERS`. Users manually add the team as a reviewer as needed.

> the behavior you described, in which the team is removed and each user in the team is added

The other users aren't added as reviewers but they are still _subscribed_ to the pull request based on the team having been added.
